lib/ansible/plugins/doc_fragments/mysql.py

```diff
@@ -19,9 +19,8 @@ class ModuleDocFragment(object):
     type: str
   login_host:
     description:
-      - Host running the database.
+      - Host running the database. Default values taken by mysqlclient (localhost or from e.g. ~/my.cnf)
```

Comment:
```suggestion
      - Host running the database. Default inherited from client library
```

Reply:
Wow, i did not know github has this function ;)

Reply:
it's quite new :)
The button to use is the document icon with `+/-` symbol, on the right of the `Preview` tab in the comment box